x-pack/plugins/features/server/oss_features.ts

```diff
@@ -28,7 +28,7 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
           app: ['discover', 'kibana'],
           catalogue: ['discover'],
           savedObject: {
-            all: ['search', 'query', 'index-pattern'],
+            all: ['search', 'query', 'index-pattern', 'background-session'],
```

Comment:
I'm not sure I understand the changes in this file... Can you help me explain what they do?

Reply:
Figures you need to give a pemission to save individual SO types. Added this with @legrego